### PR TITLE
Adds Penny

### PR DESCRIPTION
**Pull request by Alberyk**
At someone's request, adds a cat.

dumbcat

### CODE REVIEW

File: code/modules/mob/holder.dm [hunk 1]
```diff
@@ -424,6 +424,16 @@ var/list/holder_mob_icon_cache = list()
 	w_class = 1
 	item_state = "cat"
 
+/obj/item/weapon/holder/cat/penny
+	name = "\improper Penny"
```
Comment (Lohikar) [inline code/modules/mob/holder.dm]: ...why are you forcing this to be improper? This is a proper noun.

File: code/modules/mob/living/simple_animal/friendly/cat.dm [hunk 2]
```diff
@@ -286,3 +286,29 @@
 /mob/living/simple_animal/cat/kitten/Initialize()
 	. = ..()
 	gender = pick(MALE, FEMALE)
+
+
+/mob/living/simple_animal/cat/kitten
```
Comment (Lohikar) [inline code/modules/mob/living/simple_animal/friendly/cat.dm]: ...?

**Comment by NiftyBaby** [comment]
It looks so uglee.

**Comment by Lohikar** [comment]
The edges of the sprite look kind of off, makes it look almost blurr- they're transparent. Why are the edges transparent.

**Comment by Alberyk** [comment]
I will relay those concerns to the people that did the sprites.

**Comment by Pacmandevil** [comment]
It kinda looks morbidly obese, which is suiting for a cat from Central command.

**Comment by Alberyk** [comment]
The transparent borders were removed, should be less ugly now.

**Pull request merged**